bundles/org.openhab.binding.heliosventilation/src/main/resources/OH-INF/i18n/heliosventilation_de.properties

```diff
@@ -1,7 +1,7 @@
-# binding
+# add-on
 
-binding.heliosventilation.name = Helios KWL Binding
-binding.heliosventilation.description = Dies ist das Binding für Helios KWL Systeme KWL EC 200/300/500 Pro. Es benötigt eine Verbindung zum RS485 Bus für die Fernbedienung KWL-FB (9417).
+addon.heliosventilation.name = Helios KWL Binding
```

Comment:
You probably need to look at this one? I have a feeling that there should not be an 'iCloud' prefix on these lines. Or??